### ISSUE TITLE
Implement Access Policies

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/alexandria-v2#333]
When I login, access controls should be enforced for all of the levels listed on the wiki:
https://wiki.library.ucsb.edu/display/repos/ADRL+2.0+Access+Policies#ADRL2.0AccessPolicies-policies 

- [x] Restricted 
- [x] Discover Only	

~~- [ ] Campus-Only, UCSB Logins Only~~ #357 
- [x] UCSB Logins Only
- [x] UC-Community

~~- [ ] Campus-Only, Open Access~~ #357
- [x] Open Access

**Comment by val99erie** [comment]
Assume there is a group in LDAP called any_uc.  That group will be the test for the UC/shibboleth policy.  Actual Shibboleth work won't be done until the future.

**Comment by jcoyne** [comment]
Blocked by #357

**Comment by jcoyne** [comment]
Blocked by #358

**Comment by jcoyne** [comment]
Blocked by #388

**Comment by jcoyne** [comment]
Blocked by #372

**Comment by jcoyne** [comment]
This ticket involves fleshing out the AdminPolicies.

**Comment by val99erie** [comment]
Also, fill in the cancan abilities, or whatever needs to be done to enforce the AdminPolicies

**Comment by val99erie** [comment]
I don't think #388 is blocking this story.  I think it's the reverse.  #388 can't be done until we can do a query like:
```if can?(:download, record)```

So I suspect #388 is blocked by #333.

**Comment by val99erie** [comment]
I won't do the on-campus work as part of this story.  It should be covered in #357.

**Comment by mark-dce** [comment]
Updated acceptance criteria - see top
